atlas-json-parent/atlas-json-core/src/test/resources/inspect/schema/address.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "An Address following the convention of http://microformats.org/wiki/hcard",
+    "type": "object",
+    "properties": {
+        "post-office-box": { "type": "string" },
+        "extended-address": { "type": "string" },
+        "street-address": { "type": "string" },
+        "locality":{ "type": "string" },
+        "region": { "type": "string" },
+        "postal-code": { "type": "string" },
+        "country-name": { "type": "string"}
+    },
+    "required": ["locality", "region", "country-name"],
```

Comment:
I think so, at least the InstanceInspector doesn't care if it's required nor if there's any dependency in between. Possibly we may want to add it later. BTW these schema examples come from json-schema.org official - sorry I avoided to make something optimized for the tests as I'm lazy ;)
http://json-schema.org/examples.html

I submitted a follow-up PR #155 to reflect all review comments, thanks @rhuss , @zregvart !